tests/src/test/scala/cats/tests/XorTests.scala

```diff
@@ -46,6 +47,12 @@ class XorTests extends CatsSuite {
     checkAll("Eq[ListWrapper[String] Xor ListWrapper[Int]]", SerializableTests.serializable(Eq[ListWrapper[String] Xor ListWrapper[Int]]))
   }
 
+  {
+    implicit val L = ListWrapper.semigroup[String]
```

Comment:
ah, good call. this is what I get for submitting a PR at 8am before breakfast :)